#
#  sparcv9 should be 64-bit by default.
#  Upstream bug:
#    Bug 20992 - sparcv9 should be equivalent to sparc64
#    https://sourceware.org/bugzilla/show_bug.cgi?id=20992
#
--- binutils-2.27/gas/configure.tgt.orig	2016-12-06 12:06:30.740575720 +0000
+++ binutils-2.27/gas/configure.tgt	2016-12-06 12:07:11.253237475 +0000
@@ -104,6 +104,7 @@
   sparc86x*)		cpu_type=sparc arch=sparc86x  ;;
   sparclet*)		cpu_type=sparc arch=sparclet ;;
   sparclite*)		cpu_type=sparc arch=sparclite ;;
+  sparcv9*)		cpu_type=sparc arch=v9-64 ;;
   sparc*)		cpu_type=sparc arch=sparclite ;; # ??? See tc-sparc.c.
   tilegx*be)		cpu_type=tilegx endian=big ;;
   tilegx*)		cpu_type=tilegx endian=little ;;
